edb/server/compiler/compiler.py

```diff
@@ -537,6 +537,76 @@ def _compile_ql_script(
 
         return sql.decode(), argmap
 
+    def _compile_query_args(
+            self,
+            ctx: CompileContext,
+            schema: s_schema.Schema,
+            subtypes: List[Tuple[str, Any, bool]],
+    ) -> Tuple[bytes, uuid.UUID]:
+        current_tx = ctx.state.current_tx()
+
+        if not subtypes:
+            arg_ast = qlast.SelectQuery(
+                orderby=[],
```

Comment:
You shouldn't be needing to set `orderby` to `[]` explicitly.

Reply:
changed